seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/config/SourceConfig.java

```diff
@@ -65,12 +65,19 @@ public class SourceConfig extends InfluxDBConfig {
                     .noDefaultValue()
                     .withDescription("the influxdb server lower bound");
 
+    public static final Option<Integer> CHUNK_SIZE =
+            Options.key("chunk_size")
+                    .intType()
+                    .defaultValue(0)
+                    .withDescription("the influxdb client query chunk_size");
```

Comment:
**Issue**: The default value for 'chunk_size' is set to 0, which might imply that chunking is disabled by default. However, the description and practical use case of chunking suggest that it should be enabled by default to improve performance. <br> **Fix**: Consider setting a sensible default value that enables chunking by default, ensuring better out-of-the-box performance. <br> **Code Suggestion**: 
 ```
 -                    .defaultValue(0)
 +                    .defaultValue(1000)  # Assuming 1000 is a sensible default for chunk size
 ```